libdevcore/Log.cpp

```diff
@@ -43,7 +43,7 @@ void dev::simpleDebugOut(std::string const& _s, char const*)
 	static Mutex s_lock;
 	Guard l(s_lock);
 
-	cerr << _s << endl << flush;
+	cout << _s << endl << flush;
```

Comment:
logs should be going to std err by default. if you want them going elsewhere for neth, do as AZ does and register a different function in `g_logPost`.